swagger/paths_nomics.yaml

```diff
@@ -0,0 +1,65 @@
+paths:
+  "/nomics/info":
+    get:
+      description: Returns information about the exchange as a whole, and is used by Nomics to display information about Bitshares to users.
+      operationId: api.nomics.info
+      responses:
+        '200':
+          description: JSON object containing excchange info
+        '500':
+          description: Error processing parameters
+      tags:
+        - nomics
+  "/nomics/markets":
+    get:
+      description: Returns a list of the top markets on the Bitshares exchange.
+      operationId: api.nomics.markets
+      responses:
+        '200':
+          description: Top markets
+        '500':
+          description: Error processing parameters
+      tags:
+        - nomics
+  "/nomics/trades":
+    get:
+      description: Search for an asset(symbol)
+      operationId: api.nomics.trades
+      parameters:
+        - in: query
+          name: market
+          default: USD_BTS
```

Comment:
USD-BTS

Reply:
fixed in https://github.com/bitshares/bitshares-explorer-api/pull/64/commits/3ad8e65b6a97120446078c9b59efea4e1b02488a